### ISSUE TITLE
org.eclipse.jetty:jetty-jspc-maven-plugin:maven-plugin:9.4.14.v20181114 produces malformed web.xml on jdk11

### ISSUE THREAD

**Issue by bgandrew** [eclipse/jetty.project#3297]
After upgrading to `9.4.14.v20181114` and jdk11 we started to see web.xml to be broken sometimes (always on my 6-core machine) in the following manner:

```  
<servlet>
        <servlet-name>org.apache.jsp.WEB_002dINF.pages.content.edit.topic.genericTimesTopic_002d3col_jsp</servlet-name>
        <servlet-class>org.apache.jsp.WEB_002dINF.pages.content.edit.topic.genericTimesTopic_002d3col_jsp
    <servlet>
        <servlet-name></servlet-class>
    </servlet>
org.apache.jsp.WEB_002dINF.pages.content.edit.topic.topic_002dnewtabledlayout.yourMoneyTopic_002d3col_jsp</servlet-name>
        <servlet-class>org.apache.jsp.WEB_002dINF.pages.content.edit.topic.topic_002dnewtabledlayout.yourMoneyTopic_002d3col_jsp</servlet-class>
    </servlet>  
```

i.e. some urls and tags are misplaced. The amount of such errors and line numbers are different from time to time.  So probably a concurrency issue in apache jasper.

**Comment by janbartel** [comment]
Looks like this is the same as this issue reported in apache jsp:  https://bz.apache.org/bugzilla/show_bug.cgi?id=62662

That issue was fixed for apache jasper 8.5.34. Jetty-9.4.14 uses 8.5.33.  We have already updated jetty-9.4.x branch to use 8.5.35, so release jetty-9.4.15 will contain the fix.  We are discussing the timing of the next 9.4 release, so stay tuned.

**Comment by janbartel** [comment]
As its already fixed and in the release pipeline, I'll close this issue. I'll try and remember to add a comment with likely timing of the 9.4.15 release.
